src/app/shared/enum/statuses.ts

```diff
@@ -27,6 +27,16 @@ export enum StatusTitles {
   Editing = 'Відправлено на редагування',
 }
 
+export enum ProviderStatusTitles {
+  Approved = 'Заклад підтверджено',
+  Pending = 'Очікує підтвердження',
+  Editing = 'Потребує редагування'
+}
+
+export enum ProviderStatusDetails {
+  Approved = 'Ваш заклад підтверджено адміністратором. Тепер ваш заклад буде видно іншим користувачам платформи і ви зможете редагувати інформацію про заклад. '
+}
```

Comment:
I think it can be stored in the component directly because this is the only status with default text. All other cases have a reason. So I don't think that enum is appropriate in this case